addOns/requester/src/main/java/org/zaproxy/addon/requester/internal/SendHttpMessageEditorDialog.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2022 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.requester.internal;
+
+import java.awt.event.KeyEvent;
+import org.parosproxy.paros.extension.ExtensionHook;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.extension.httppanel.Message;
+import org.zaproxy.zap.extension.requester.ExtensionRequester;
+import org.zaproxy.zap.extension.requester.ManualHttpRequestEditorPanel;
+import org.zaproxy.zap.view.ZapMenuItem;
+
+public class SendHttpMessageEditorDialog extends AbstractHttpMessageEditorDialog {
+
+    private static final long serialVersionUID = 1L;
+
+    public SendHttpMessageEditorDialog(ManualHttpRequestEditorPanel panel) {
+        super("manReq.dialog.title", panel);
+    }
+
+    @Override
+    public void load(ExtensionHook extensionHook) {
+        super.load(extensionHook);
+
+        ZapMenuItem menuItem =
+                new ZapMenuItem(
+                        "menu.tools.manReq",
+                        View.getSingleton().getMenuShortcutKeyStroke(KeyEvent.VK_M, 0, false));
```

Comment:
Doesn't the requester use `ctrl+w`?

Reply:
This replaces the core Manual Request Editor, which uses other shortcut.